doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_smp.rst

```diff
@@ -0,0 +1,224 @@
+.. _ug_nrf54h20_suit_smp:
+
+SUIT and SMP protocol (including SUIT SMP extension)
+####################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Adopting SUIT as a firmware update method offers a flexible and extensible update process.
+This document illustrates how the Zephyr management subsystem, via the SMP protocol, can be leveraged to manage SUIT-based images alongside existing MCUBoot images, providing an efficient approach to firmware management.
+
+Slots and Images in Zephyr
+==========================
+
+In the nRF Connect SDK, *slot* and *image* concepts are derived from MCUboot.
+An *image* consists of two slots: a *primary* and a *secondary*.
+The application runs from the primary slot, while updates are uploaded to the secondary slot.
+MCUboot is responsible for swapping these slots during boot.
+
+The slots support a single upgradable application, with equal sizes.
+Zephyr supports up to two images, each represented by an index.
+
+.. note::
+   Image metadata, such as version information, is embedded within the image payload.
+
+Components in SUIT
+==================
+
+SUIT defines a *component* as an updatable logical block, which can be firmware, software, configuration, or data.
+This is similar to how MCUmgr defines an *image*.
+Updates are delivered to the device as a SUIT *envelope,* containing SUIT manifests and, optionally, integrated payloads.
+The relationships between components are maintained through SUIT manifests.
+
+The SUIT envelope serves as the main metadata source for the payload.
+Adding metadata inside the image can lead to synchronization issues, as the lifecycles of the envelope and image may diverge.
+Therefore, manage metadata at the envelope level.
```

Comment:
Lines 35 to 37... looks strange for me. If it is intended to compare SUIT to MCUboot/Zephyr, I would consider something like:

"Metadata, such a version information, expected image digest, image size is embedded into SUIT manifest"